src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.scss

```diff
@@ -0,0 +1,11 @@
+@media screen and (max-width: 1870px) {
```

Comment:
Is one of these redundant?

Reply:
are there predefined govuk breakpoints? e.g. `@include govuk-media-query($until: desktop)`